internal/installcfg/installcfg.go

```diff
@@ -235,6 +236,9 @@ func GetInstallConfig(log logrus.FieldLogger, cluster *common.Cluster) ([]byte,
 	if err != nil {
 		return nil, err
 	}
+	if addRhCa {
+		cfg.AdditionalTrustBundle = fmt.Sprintf(` | %s`, ca)
```

Comment:
@rollandf can u open a task on @carbonin to sypport append of some fields, and then also adjust this code?